packages/block-library/src/latest-posts/edit.js

```diff
@@ -227,68 +233,135 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 	const hasPosts = !! latestPosts?.length;
 	const inspectorControls = (
 		<InspectorControls>
-			<PanelBody title={ __( 'Post content' ) }>
-				<ToggleControl
-					__nextHasNoMarginBottom
+			<ToolsPanel
+				label={ __( 'Post content' ) }
+				resetAll={ () =>
+					setAttributes( {
+						displayPostContent: false,
+						displayPostContentRadio: 'excerpt',
+						excerptLength: DEFAULT_EXCERPT_LENGTH,
+					} )
+				}
+				dropdownMenuProps={ dropdownMenuProps }
+			>
+				<ToolsPanelItem
+					hasValue={ () => !! displayPostContent }
 					label={ __( 'Post content' ) }
-					checked={ displayPostContent }
-					onChange={ ( value ) =>
-						setAttributes( { displayPostContent: value } )
+					onDeselect={ () =>
+						setAttributes( { displayPostContent: false } )
 					}
-				/>
+					isShownByDefault
+				>
+					<ToggleControl
+						__nextHasNoMarginBottom
+						label={ __( 'Post content' ) }
+						checked={ displayPostContent }
+						onChange={ ( value ) =>
+							setAttributes( { displayPostContent: value } )
+						}
+					/>
+				</ToolsPanelItem>
 				{ displayPostContent && (
-					<RadioControl
+					<ToolsPanelItem
+						hasValue={ () => !! displayPostContentRadio }
```

Comment:
```suggestion
						hasValue={ () => displayPostContentRadio !== 'excerpt' }
```

Sorry, my previous feedback was incorrect. Here too, we need to check the default value defined in block.json.